challenges/towers_of_hanoi/towers_of_hanoi.py

```diff
@@ -0,0 +1,13 @@
+
+def towers_of_hanoi(n, peg_1, peg_3, peg_2):
```

Comment:
This does not match the function designated in the spec. The spec calls for a function `towers_of_hanoi(n)`, which only has one parameter. This function works fine, just write an additional function that starts the recursion with the proper parameters.